app/index.js

```diff
@@ -7,7 +7,13 @@ export {Positioner} from './scripts/positioner';
 
 // TypeScript exports
 // Must contain the same exports as in '../app-typescript/index'
-export {HelloWorld} from '../app-typescript/index';
+export {
+    HelloWorld,
+    Button,
+    Label,
+    Badge,
+    Alert,
```

Comment:
Add TagLabel

Reply:
In agreement with @fritzSF, we are removing that component